### ISSUE TITLE
Ember g route doesn't look for a router.ts file

### ISSUE THREAD

**Issue by oliverlj** [ember-cli/ember-cli#9641]
Thank you for taking the time to open an issue!

For bug reports please include the following section in your issue details.
If you include instructions on how to reproduce the bug or a failing test case
it will make it easier for us to track down the issue you're having.

❯ ember g route six-accounts
installing route
  create app/routes/six-accounts.ts
  create app/templates/six-accounts.hbs
ENOENT: no such file or directory, open '/home/oliver/git/fritzy/front/fz-front-simulator/app/router.js'

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
❯ ember version --verbose && npm --version && yarn --version
ember-cli: 3.28.0
node: 14.17.6
v8: 8.4.371.23-node.76
uv: 1.41.0
zlib: 1.2.11
brotli: 1.0.9
ares: 1.17.2
modules: 83
nghttp2: 1.42.0
napi: 8
llhttp: 2.1.3
openssl: 1.1.1l
cldr: 39.0
icu: 69.1
tz: 2021a
unicode: 13.0
os: linux x64
7.20.3
1.22.11
```

**Comment by bertdeblock** [comment]
Are you using [ember-cli-typescript-blueprints](https://github.com/typed-ember/ember-cli-typescript-blueprints)?
If so, I think the `route` blueprint that is used is coming from that package and not from `ember-source`.

I assume official support for a `router.ts` file would be part of https://github.com/emberjs/rfcs/pull/724.

**Comment by oliverlj** [comment]
thanks for pointing this addon, i discover it from your comment :)

**Comment by oliverlj** [comment]
Yes i'm using ember-cli-typescript-blueprints. should i create an issue there instead of here ?

**Comment by bertdeblock** [comment]
You could create an issue there as well I think. Both blueprints will have the same issue as they always look for a `router.js` file.
